solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureLtrScoringModel.java

```diff
@@ -46,8 +46,8 @@ public void getInstanceTest() throws FeatureException
   {
 
     final Map<String,Object> map = new HashMap<String,Object>();
-    map.put(ManagedFeatureStore.NAME_KEY, "test");
-    map.put(ManagedFeatureStore.CLASS_KEY, OriginalScoreFeature.class.getCanonicalName());
+    map.put("name", "test");
```

Comment:
+1, in general I think its much better to have static global strings than local magic strings. Makes things easier to change if needed, provides an in house explanation of what this string is, etc.